es/criteria/_template.md

```diff
@@ -6,7 +6,7 @@
 
 ## Qué no hace
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks.